src/pages/upgrades/get-involved/index.tsx

```diff
@@ -163,7 +163,44 @@ const StyledCalloutBanner = styled(CalloutBanner)`
   }
 `
 
-const GetInvolvedPage = ({ data, location }) => {
+type Layer = "el" | "cl"
+type Client = {
+  name: string
+  background: string
+  description: JSX.Element
+  alt: string
+  url: string
+  image: (isDarkTheme?: boolean) => string
+  githubUrl: string
+  isBeta?: boolean
+  children?: React.ReactNode
+}
+type Clients = {
+  [key in Layer]: Client[]
+}
+
+type ImageNames =
+  | "besu"
+  | "erigon"
+  | "geth"
+  | "nethermind"
+  | "prysm"
+  | "lighthouseDark"
+  | "lighthouseLight"
+  | "tekuLight"
+  | "tekuDark"
+  | "lodestar"
+  | "nimbus"
+  | "rhino"
+
+type Images = { [key in ImageNames]: string }
+
+type IProps = {
+  bountyHunters: {
+    nodes: Person[]
+  }
+} & Images
+const GetInvolvedPage = ({ data, location }: PageProps<IProps>) => {
```

Comment:
In page components, we don't need to specify the `IProps` since gatsby is auto-generating the types for us.

You can check it [here](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/typescript.md#migration-guide--contribution-opportunity-) for more context on how to do it.

Basically, you need to 
1. Give a name to the exported query. In this case `query GetInvolvedPage {`
2. Run `yarn start`. Gatsby is going to generate the types for this named query
3. Define the props for this page like this `PageProps<Queries.GetInvolvedPageQuery, Context>`

Let me know if it makes sense. If not, I can jump in and make those changes myself 👍🏼